client/components/Nav.jsx

```diff
@@ -263,7 +263,7 @@ class Nav extends React.PureComponent {
               }}
             >
               <span className="nav__item-header">File</span>
-              <InlineSVG className="nav__item-header-triangle" src={triangleUrl} />
+              <InlineSVG className="nav__item-header-triangle" src={triangleUrl} title="File" />
```

Comment:
I don't think this SVG icon should have a title, since it doesn't have an action! The button that is holding the icon is the thing that's important, and I think the title here is superfluous. Same for the rest of the menu items.